L1Trigger/CSCTriggerPrimitives/interface/CSCAnodeLCTProcessor.h

```diff
@@ -123,8 +123,9 @@ class CSCAnodeLCTProcessor : public CSCBaseboard {
   std::vector<CSCALCTPreTriggerDigi> thePreTriggerDigis;
 
   /* data members for high multiplicity triggers */
-  void encodeHighMultiplicityBits();
+  void encodeHighMultiplicityBits(const std::vector<int> wire[CSCConstants::NUM_LAYERS][CSCConstants::MAX_NUM_WIRES]);
```

Comment:
@dildick , it seems, that reference to this vector would be better than vector itself.

Reply:
@civanch correct. I will fix it.